src/H5Ppublic.h

```diff
@@ -1582,16 +1582,12 @@ H5_DLL int H5Piterate(hid_t id, int *idx, H5P_iterate_t iter_func, void *iter_da
  *          property is being closed. The #H5P_prp_close_func_t callback
  *          function is defined as follows:
  *
- *          \snippet this H5P_prp_cb2_t_snip
+ *          \snippet this H5P_prp_cb1_t_snip
```

Comment:
The 'cb1' / 'cb2' types are just convenience / maintenance helpers in the header file - the documentation should use the types for the parameters in the H5P* routines.

Reply:
e.g. H5P_prp_create_func_t, H5P_prp_set_func_t, etc.